tests/unit/test_environment.py

```diff
@@ -49,6 +49,7 @@ def test_step_is_running():
         pipeline_run_id="run_id",
         step_name="step",
         cache_enabled=True,
+        pipeline_requirements=["zenml"],
```

Comment:
Why is this needed? Shouldn't line 710 of steps/utils.py already take care of this?

Reply:
I don't think it does because that's just for testing of StepEnvironment, it tries to create a new object from that class. also, that list could be just empty rather than have "zenml" on it!

Reply:
I think an empty list would be nicer indeed just to show that this is a "dummy" value.